spec/game.test.js

```diff
@@ -0,0 +1,38 @@
+var Game = require("../docs/game");
+var Frame = require("../docs/frame");
+
+describe("Game", function() {
+  
+  it("adds a completed frame to the frames array", function(){
+    var game = new Game();
+    game.enterFrame(1,2);
+    expect(game.getFrames()).toEqual([[1,2]]);
+  });
+
+  it("adds the total of the rolls in a frame to the rollsTotal array", function(){
+    var game = new Game();
+    game.enterFrame(1,2);
+    expect(game.getRollsTotals()).toEqual([3]);
+  });
+
+  it("adds the Strike bonus to the bonuses array", function(){
+    var game = new Game();
+    game.enterFrame(10,0);
+    game.enterFrame(1,2);
+    expect(game.getBonuses()).toEqual([3,0,0,0,0,0,0,0,0,0,0,0]);
+  });
+
```

Comment:
this test seems well written to me, the bonus position correlates to which frame the bonus adds to. will pass when you've got get bonuses to work correctly. Not sure if there might be some testing of state here rather than behaviour but I don't really get what defines testing state so can't be sure, something for me to look into